doc/modules/model_evaluation.rst

```diff
@@ -2185,49 +2185,46 @@ of 0.0.
     -0.552
 
 
-|details-start|
-**D2 Brier score**
-|details-split|
+.. dropdown:: **D2 Brier score**
 
-The :func:`d2_brier_score` function implements the special case
-of D² with the Brier score, see :ref:`brier_score_loss`, i.e.:
+   The :func:`d2_brier_score` function implements the special case
+   of D² with the Brier score, see :ref:`brier_score_loss`, i.e.:
 
-.. math::
+  .. math::
 
-  \text{dev}(y, \hat{y}) = \text{brier_score_loss}(y, \hat{y}).
+    \text{dev}(y, \hat{y}) = \text{brier_score_loss}(y, \hat{y}).
 
-This is also referred to as the Brier Skill Score (BSS).
+  This is also referred to as the Brier Skill Score (BSS).
```

Comment:
do these spaces actually make a difference in the rendered docs? This makes it seem like you re-wrote the whole section may be add confusion to track changes/version control